Mathlib/Analysis/Complex/HasPrimitives.lean

```diff
@@ -264,6 +267,16 @@ theorem IsConservativeOn.isExactOn_ball (hf' : ContinuousOn f (ball c r))
     IsExactOn f (ball c r) :=
   ⟨fun z ↦ wedgeIntegral c z f, fun _ ↦ hf.hasDerivAt_wedgeIntegral hf'⟩
 
+theorem isConservativeOn_iff_isDifferentiableOn
+    {U : Set ℂ} (hU : IsOpen U) (hf' : ContinuousOn f U) :
+    IsConservativeOn f U ↔ DifferentiableOn ℂ f U := by
```

Comment:
Can't you say that `ContinuousOn` and `IsConservativeOn` are equivalent to `DifferentiableOn`? Seems a bit funny otherwise to use the direction where you have to both input differentiability and continuity?

Reply:
Oh good point, what was I thinking!? Done!